### ISSUE TITLE
TF 2.3: Learning rate is NOT printed in the training output when using "ReduceLROnPlateau" callback

### ISSUE THREAD

**Issue by mikemikezhu** [tensorflow/tensorflow#43385]
**TensorFlow version:** 2.3.0
**Python version:** 3.8.2
**Issue:**
Given the TensorFlow version is upgraded to 2.3.0,
Given using "ReduceLROnPlateau" callback,
When perform training on the model with learning rate decay,
Then learning rate "lr - xxx" is NOT printed in the training output

```
reduce_learning_rate = ReduceLROnPlateau(monitor='val_loss', factor=0.9, patience=5)
history = model.fit(x_train, y_train, epochs=training_epochs, batch_size=batch_size, validation_data=(x_test, y_test), callbacks=[reduce_learning_rate])
```

**TensorFlow 2.2.0:** Showing learning rate in the output of each epoch
`133/133 [==============================] - 2s 15ms/step - loss: 6.5944 - accuracy: 0.5781 - val_loss: 0.1361 - val_accuracy: 0.9640 - lr: 0.0200`

**TensorFlow 2.3.0:** Not showing learning rate in the output of each epoch
`133/133 [==============================] - 2s 15ms/step - loss: 6.5944 - accuracy: 0.5781 - val_loss: 0.1361 - val_accuracy: 0.9640`

**Comment by ravikyram** [comment]
@mikemikezhu 

Can you share colab link or simple standalone code to reproduce the issue in our environment.It helps us in localizing the issue faster. Thanks!
